networking/socket/fuzz/metadata

```diff
@@ -1,11 +1,12 @@
 [General]
-name=/kernel/networking/route/route_func
+name=networking/route/route_func
 owner=Hangbin Liu <haliu@redhat.com>
 description=A socket testing framework for all protocols
 license=GPLv2
 confidential=no
 destructive=no
 
 [restraint]
-entry_point=make run
-repoRequires=networking/common;cki_lib
+entry_point=bash ./runtest.sh
+dependencies=gcc;lksctp-tools-devel
+repoRequires=cki_lib
```

Comment:
This seems needing  networking/common  on repoRequires. 
The test runtest.sh  has code for sourcing networking/common:
. ../../common/include.sh

Reply:
Updated.